hc/api/tests/test_sendalerts.py

```diff
@@ -30,13 +30,14 @@ def test_it_handles_few(self, mock):
         assert set(names) == set(handled_names)
         ### The above assert fails. Make it pass
 
+
     def test_it_handles_grace_period(self):
         check = Check(user=self.alice, status="up")
         # 1 day 30 minutes after ping the check is in grace period:
         check.last_ping = timezone.now() - timedelta(days=1, minutes=30)
         check.save()
 
         # Expect no exceptions--
-        Command().handle_one(check)
-
+        result = Command().handle_one(check)
     ### Assert when Command's handle many that when handle_many should return True
+        assert result is True
```

Comment:
you asserted ommand's handle_one but you should have asserted command  handle_many instead